plugins/item_tasks/plugin_tests/tasks_test.py

```diff
@@ -103,12 +103,16 @@ def testAddItemTasksToFolderFromJson(self):
         with open(os.path.join(os.path.dirname(__file__), 'spec.json')) as f:
             spec = f.read()
         parsedSpec = json.loads(spec)
+
+        token = self.model('token').createToken(
+            user=self.admin, scope='item_task.set_task_spec.%s' % folder2['_id'])
```

Comment:
I generally avoid negative testing specific endpoints, instead I'd rather rely on our standard security protocols and just have them well-tested in isolation. These cases are exceptions to normal authorization controls, so maybe we want to add more negative testing for them, but if so, let's do it on a separate topic.